documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -2720,6 +2706,16 @@ __OGM default configuration__ is transactional.
 We always recommend using transactional caches, at least unless you are confident that data consistency is not important for your use case.
 ====
 
+==== Remote Query Capability
+
+Hibernate OGM backend is able to run the queries it needs to materialize relations.
+From version __5.4.0.Alpha2__ the dialect is capable to translate JPQL or Criteria queries to the Infinispan remote queries.
```

Comment:
It was preexisting but it should be either:
 `to the **corresponding** Infinispan remote queries`
or
 `to Infinispan remote queries`

Reply:
Criteria?